manifests/service.pp

```diff
@@ -49,4 +49,66 @@
                   ],
   }
 
+  # if the Sentry config changes, do a full restart of the Sentry Celery workers
+  exec { 'restart-sentry-worker':
+    command     => '/usr/bin/systemctl stop sentry-worker; /usr/bin/systemctl start sentry-worker',
+    path        => '/bin:/usr/bin',
+    refreshonly => true,
+    subscribe   => Class['::sentry::config'],
+  }
+
+  # Sentry Celery Beat
+  file { '/etc/systemd/system/sentry-beat.service':
+    ensure  => present,
+    mode    => '0644',
+    owner   => 'root',
+    group   => 'root',
+    content => template('sentry/sentry-beat.service.erb'),
+    notify  => Exec['enable-sentry-services'],
+  }
+
+  service { 'sentry-beat':
```

Comment:
We're currently launching the beat process from the main worker service by passing the `-B` argument: https://github.com/covermymeds/puppet-sentry/blob/master/templates/sentry-worker.service.erb#L13.  Need to reconcile this as running more than 1 beat sounds bad.

```
/var/lib/sentry/bin/celery worker --help
Usage: celery worker [options] 
...
...
  -B, --beat            Also run the celery beat periodic task scheduler.
                        Please note that there must only be one instance of
                        this service.
...
...
```